openpype/lib/applications.py

```diff
@@ -973,6 +973,9 @@ def __init__(
 
         self.env_group = env_group
 
+        stdout = data.pop("stdout", None)
```

Comment:
I think this file can stay untouched and as was, modified lines actually added only confusion (pop stdout/stderr and then assign it back).

Reply:
This is coming from #5903 where you've picked up on the same thing. Moving the conversation to there.